src/pages/RefundExternal.tsx

```diff
@@ -0,0 +1,255 @@
+                required
+                type="file"
+                id="refundUpload"
+                data-testid="refundUpload"
+                accept="application/json,image/png,imagine/jpg,image/jpeg"
+                onChange={(e) => uploadChange(e)}
+            />
+            <Show when={refundInvalid() !== undefined}>
+                <hr />
+                <button class="btn" disabled={true}>
+                    {(() => {
+                        switch (refundInvalid()) {
+                            case RefundError.InvalidData:
+                                return t("invalid_refund_file");
+                        }
+                    })()}
+                </button>
+            </Show>
+            <hr />
+            <RefundButton swap={refundJson} setRefundTxId={setRefundTxId} />
+            <Show when={refundTxId() !== ""}>
+                <hr />
+                <p>{t("refunded")}</p>
+                <hr />
+                <BlockExplorer
+                    typeLabel={"refund_tx"}
+                    asset={refundJson().asset || refundJson().assetSend}
+                    txId={refundTxId()}
+                />
+            </Show>
+            <SettingsMenu />
+        </>
+    );
+};
+
+const RefundRsk = () => {
+    const { t } = useGlobalContext();
+    const { signer, getEtherSwap } = useWeb3Signer();
+
+    const [logRefundableSwaps, setLogRefundableSwaps] = createSignal<
+        LogRefundData[]
+    >([]);
+    const [refundScanProgress, setRefundScanProgress] = createSignal<
+        string | undefined
+    >(undefined);
+
+    let refundScanAbort: AbortController | undefined = undefined;
+
+    onCleanup(() => {
+        if (refundScanAbort) {
+            refundScanAbort.abort();
+        }
+    });
+
+    // eslint-disable-next-line solid/reactivity
+    createEffect(async () => {
+        setLogRefundableSwaps([]);
+
+        if (refundScanAbort !== undefined) {
+            refundScanAbort.abort("signer changed");
+        }
+
+        if (signer() === undefined) {
+            return;
+        }
+
+        setRefundScanProgress(
+            t("logs_scan_progress", {
+                value: Number(0).toFixed(2),
+            }),
+        );
+
+        refundScanAbort = new AbortController();
+
+        const generator = scanLogsForPossibleRefunds(
+            refundScanAbort.signal,
+            signer(),
+            getEtherSwap(),
+        );
+
+        for await (const value of generator) {
+            setRefundScanProgress(
+                t("logs_scan_progress", {
+                    value: (value.progress * 100).toFixed(2),
+                }),
+            );
+            setLogRefundableSwaps(logRefundableSwaps().concat(value.events));
+        }
+
+        setRefundScanProgress(undefined);
+    });
+
+    return (
+        <Show
+            when={import.meta.env.VITE_RSK_LOG_SCAN_ENDPOINT !== undefined}
+            fallback={<p>{t("rsk_log_endpoint_not_available")}</p>}>
```

Comment:
Let's not show this in the UI, because there is nothing that can be done about it when you open the page. A  `log.warn` should be enough